test/Interop/Cxx/reference/Inputs/virtual-methods-with-rvalue-reference.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
I'd probably put this under `Interop/Cxx/foreign-reference` where the rest of the FRT tests are, but I don't feel strongly about it 😉

Reply:
Yeah, I think it is equally good in both places. The CI is already running so I'll move it if it fails and merge as is if it is passing.

Reply:
There was a small windows specific failure so I ended up moving the files